src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,719 @@
+{{% /notice %}}
+
+## Use Cases
+
+- Request or update device state configurations
+  - _I want to track & control smart home devices from my mobile application_
+  - _I want to remotely control household heater by turning them on/off or by setting up desired temperature_ 
+- Request device actions based on some commands
+  - _I want to remotely unlock door for a family visitor_ 
+  - _I want to remotely instruct device to do some action on basis of a command_
+
+## Reference Architecture
+
+Following section talks about architecture considerations for command & control using IoT Core device shadow
+ - When requesting update to device state configuration
+ - When requesting device actions based on some commands  
+
+{{< tabs groupId="MQTT-arch">}}
+{{% tab name="State Control" %}}
+
+# ![Track and control device state configuration using shadow ](architecture1.svg)
+
+
+- _AWS IoT Core_ is the MQTT message broker processing messages
+- _Device_ is the IoT thing to be controlled
+- _Application_ is the remote logic that issues commands, update device state configurations and consumes device telemetry data
+- _Device Shadow_ is the replica which makes device's state available to applications and other services
+- _ShadowTopicPrefix_ is an abbreviated form of the topic which can refer to either a named or an unnamed shadow, referred for `$aws/things/device1/shadow` (classic shadow) in explanation below.
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint, and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) for get & update shadow event operations. Also, instead of subscribing individually to each reserved shadow topic below, common prefix could be used for generalised subscription like `ShadowTopicPrefix/get/#` , `ShadowTopicPrefix/update/#`
+
+- `ShadowTopicPrefix/get/accepted`
+- `ShadowTopicPrefix/get/rejected`
+- `ShadowTopicPrefix/update/accepted`
+- `ShadowTopicPrefix/update/rejected`
+- `ShadowTopicPrefix/update/delta`
+
+2. On successful connect with _AWS IoT Core_ endpoint & on successful subscription to reserved shadow topics, the _Device_ publish request to `ShadowTopicPrefix/get` topic and process latest shadow file received on `ShadowTopicPrefix/get/accepted` subscribed topic. After processing _desired_ state or configuration changes from the shadow file, _Device_ makes publish request to `ShadowTopicPrefix/update` topic with device current state as _reported_ data on shadow file.     
```

Comment:
On _a_ successful
with _the_ AWS IoT Core
publishes a request
topic, and processes the latest shadow file
on _the_ ShadowTopic
After processing _the_

Reply:
Corrected